samcli/commands/build/build_context.py

```diff
@@ -110,62 +113,67 @@ def _setup_build_dir(build_dir, clean):
         return str(build_path.resolve())
 
     @property
-    def container_manager(self):
-        return self._container_manager
+    def container_manager(self) -> ContainerManager:
+        # Note(xinhol): self._container_manager will be assigned with a value if it is None in __enter__()
+        return self._container_manager  # type: ignore
 
     @property
-    def function_provider(self):
-        return self._function_provider
+    def function_provider(self) -> SamFunctionProvider:
+        # Note(xinhol): self._container_manager will be assigned with a value if it is None in __enter__()
```

Comment:
typo in the comment? _function_provider instead of _container_manager?

Reply:
Also, what is the value of the comment in general?

Reply:
good catch!